cve_bin_tool/checkers/libevent.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+
+CVE checker for Libevent
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-15590/product_id-32303/Libevent-Project-Libevent.html
+
+"""
+
+from cve_bin_tool.checkers import Checker
+
+
+class LibeventChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"Added a signal to event base %p with signals already added to event_base %p.  Only one can have signals at a time with the %s backend.  The base with the most recently added signal or the most recent event_base_loop() call gets preference; do not rely on this behavior in future Libevent versions.",
+        r"Error from libevent when adding event"
+        r"Error from libevent when adding event for DNS server",
+        r"Error from libevent when adding timer event",
+    ]
+
+    FILENAME_PATTERNS = [
+        r"libevent.so",
+    ]
+
+    VERSION_PATTERNS = [
+        r"([0-9]\.[0-9]\.[0-9]{1,2})-stable",
+        r"([0-9]\.[0-9]\.[0-9]+)\nUnknown\nEBMLVoid",  # This string may be brittle to changes in string ordering
```

Comment:
If I remove this string, version pattern throws an error. So I don't thin so that it can be removed.

Reply:
Ok, so add this as an additional version pattern, that way it works from what I tried.

Reply:
This works at my system, @terriko could you please help us out with this issue. And test this if it works or not.

Reply:
The second version pattern can't be used as this is a copy/paste from libebml checker:

https://github.com/intel/cve-bin-tool/blob/383c26d1c9fd41ddb6f8add208c0b99b01abeaca/cve_bin_tool/checkers/libebml.py#L20

So this will raise false positives

Reply:
> The second version pattern can't be used as this is a copy/paste from libebml checker:
> 
> https://github.com/intel/cve-bin-tool/blob/383c26d1c9fd41ddb6f8add208c0b99b01abeaca/cve_bin_tool/checkers/libebml.py#L20
> 
> So this will raise false positives

Actually, I was searching for a pattern that will go with the deb file. And this was the only one that was working. So I just copied from there.